domain-models-interface-extensions/src/main/java/org/folio/rest/tools/GenerateRunner.java

```diff
@@ -166,6 +166,7 @@ public void generate(String inputDirectory) throws Exception {
     int numMatches = 0;
 
     File []ramls = new File(inputDirectory).listFiles( (dir, name) -> name.endsWith(".raml") );
+    Arrays.sort(ramls);
```

Comment:
Is sorting of raml files sufficient? I mean if I declared two public functions in two files a.java and b.java with this it will magically work if a sorts before b. It might be determistic, but in no way logical. Eg if I move function a.java to q.java it would work again...

Reply:
Yes, this does not solve the cause of the problem.
However, it either always fails the build or it always succeeds.
If it fails it will also fail on the developer local machines and enables them to debug and fix it.